src/js/components/support-button/SupportButton.jsx

```diff
@@ -1,39 +1,47 @@
 import React, { useEffect } from 'react';
 
+import { faLifeRing } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
 import PropTypes from 'prop-types';
 import { useChat } from 'react-live-chat-loader';
 
 import Translate from 'utils/Translate';
 
 import './SupportButton.scss';
 
-const SupportButton = ({ buttonColor, text, defaultText }) => {
+const SupportButton = ({ text, defaultText }) => {
   const [, loadChat] = useChat();
+
   useEffect(() => {
-    loadChat({ open: false });
+    loadChat({ open: false }); // instantiate `window` object
+    axios.get('/openboxes/api/helpscout/configuration/')
+      .then((response) => {
+        window.Beacon('config', response.data);
+      });
   }, []);
 
   const toggleOpenChat = () => window.Beacon('toggle');
+
   return (
     <button
       type="button"
-      className="btn ml-1 btn-light"
-      style={{ backgroundColor: buttonColor }}
+      className="btn btn-helpscout ml-1 mr-1"
       onClick={toggleOpenChat}
     >
+      <FontAwesomeIcon icon={faLifeRing} />
+      &nbsp;
       <Translate id={text} defaultMessage={defaultText} />
     </button>
   );
 };
 
 SupportButton.propTypes = {
-  buttonColor: PropTypes.string,
   text: PropTypes.string,
   defaultText: PropTypes.string,
 };
 
 SupportButton.defaultProps = {
-  buttonColor: undefined,
   text: 'Help',
   defaultText: 'Help',
```

Comment:
add 
```
locale: PropTypes.string.isRequired,
```

Reply:
sorry for the spam, I wanted to help you, but it was the first time I tried to do it via GitHub and didn't know it would collapse to a few single reviews.